themes/custom/scaffold/scss/wysiwyg.scss

```diff
@@ -0,0 +1,17 @@
+// Make sure the charset is set appropriately
+@charset "UTF-8";
+
+@import "global";
+@import "foundation-sites/scss/foundation";
+
+// Put remote font imports here.
+// @import url($google-fonts);
+
+@include foundation-everything();
```

Comment:
I don't think we actually want all of foundation here.  I think we just want a few bits.  Maybe something like:

`@include foundation-typography;
@include foundation-button;
@include foundation-table;
@include foundation-thumbnail;
@include foundation-visibility-classes;
@include foundation-float-classes;`

Of course, that's all overrideable on a per-project basis.